averages.js

```diff
@@ -0,0 +1,51 @@
+var _ = require("lodash");
+var pry = require('pryjs');
+
+function total(array) {
+	total = 0;
```

Comment:
Are you using tabs or spaces for you tab character? It's a small thing, but would be worth getting it to spaces to avoid merge conflicts in the future.